### ISSUE TITLE
[469d] Inconsistent player head bump velocity

### ISSUE THREAD

**Issue by owythings** [OldUnreal/UnrealTournamentPatches#1571]
Request for an enhancement/fix for inconsistent physics:

When player bumps against a (sloped) ceiling they get a downwards boost in velocity. (and sideways if sloped)

However this boost seems to be random. Sometimes the player gets a big boost and can take damage, and sometimes barely any boost at all.
FPS seems to have an effect, lower FPS (<60) seems to increase the chance of bigger boosts.

Would it be possible to fix/enhance the head bump physics?
For example a fixed multiplier such as `Velocity.Z *= -0.5` would be a good consistent amount.

**Comment by BerserkerBG** [comment]
Yes, that is indeed true. This issue needs an example map. Here is an example with a sloped ceiling. Notice that every time I land on a different color, behavior is definitely random (my fps is also consistent in this example):
https://github.com/OldUnreal/UnrealTournamentPatches/assets/58362332/12629c07-0d16-4d80-87ae-a24cf56db36c

.unr Map file:
[CeilingSlopeBump.zip](https://github.com/OldUnreal/UnrealTournamentPatches/files/15251698/CeilingSlopeBump.zip)

This velocity issue also happens on other types of ceilings as well. For example, behavior is random on flat ceilings too - when you bump your head on a flat ceiling with high velocity its supposed to boost you down with the same momentum, but sometimes the momentum is cancelled and you are not boosted downwards.
